Formula/watchman.rb

```diff
@@ -23,7 +29,8 @@ def install
                           "--with-pcre",
                           # we'll do the homebrew specific python
                           # installation below
-                          "--without-python"
+                          "--without-python",
+                          "--enable-statedir=#{var}/run/watchman"
```

Comment:
This is what you get by the end of `install` if neither `/usr/local/var/run` nor `/usr/local/var/run/watchman` exist at the beginning:
```
$ ls -ld /usr/local/var/run
drwxr-xr-x  3 joe  admin  102 Jun  9 06:56 /usr/local/var/run
$ ls -ld /usr/local/var/run/watchman
drwxrwsrwx  2 joe  admin  102 Jun  9 06:56 /usr/local/var/run/watchman
```